File: UnityPlugin/src/main/java/ir/FiroozehCorp/GameService/UnityPackage/App/Interfaces/IGameServiceLoginCheck.java
package ir.FiroozehCorp.GameService.UnityPackage.App.Interfaces;

/**
 * Created by Alireza Ghodrati on 09/10/2018 in
 * Time 09:58 AM for App ir.firoozeh.unitymodule.Interfaces
 */

public interface IGameServiceLoginCheck {
    void isLoggedIn (boolean Status);
    void OnError (String Error);
}

File: unitymodule/src/main/java/ir/firoozeh/unitymodule/Interfaces/IGameServiceLoginCheck.kt
package ir.firoozeh.unitymodule.Interfaces

/**
 * Created by Alireza Ghodrati on 09/10/2018 in
 * Time 09:58 AM for App ir.firoozeh.unitymodule.Interfaces
 */

interface IGameServiceLoginCheck {
    fun isLoggedIn(Status: Boolean)

    fun OnError(Error: String)
}
